### DISCUSSION TITLE
Webos App Missing features?

### DISCUSSION THREAD

**Discussion by RayneRPT** [redphx/better-xcloud#463] [General]
Well I managed to install better xcloud on my OLED B36LA 77" but I'm having some issues. Mention that I don't have root on my TV since it will void Warranty. 

Can't configure a lot of the settings that are on the android app look at this photo I took. I can't even set my preferred nearest stream server that is Western Europe and not UK.

Streaming settings, OpenGL renderer gets me stream black screen.

Is it like this on the WebOS app? 
Thank you
![20240725_213915](https://github.com/user-attachments/assets/27c694b9-eaef-4b51-b2a0-abe5f26e23ca)

**Comment by redphx** [comment]
Try again with 5.5.0

**Reply by RayneRPT** [reply]
Thanks for helping, unfortunately all I get now with this new version is this when I press the flag:
![20240730_220015](https://github.com/user-attachments/assets/e82c21a7-071e-4b8a-88b0-4ef5f6717523)
On left side menu when I press Better Xcloud I get to this same menu and now there's nothing to configure like stream renderer not even controller vibration only shows that has you can see from the picture.

By the way when I press the better xcloud button it doesn't go straight to the menu I have to work around some buttons like previous or something for it to show. 

Quit App button doesn't work also.

Since I'm trying to help the development I'm pointing out another problem, TV screen saver kicks in still if you are using the game pad when navigating library Im using PS5 dualsense. 

Better xcloud menu is junky using gamepad it starts to bounce between the xcloud menu and the game library.

Pressing the version number leaves to the github page were you can't scroll down nor leave the page and now I'm stuck on the page. Even if I go to TV menu and start the App again. ( fixed, press and hold back button on TV remote force closes the App). 

I feel a tiny bit more input lag than using TV browser app.

Edit: trying to search for a game Using 2.4ghz keyboard doesn't work but also the virtual keyboard from TV don't show up.

I can be stupid the question maybe but do I need to disable developer mode?

Thanks for this app by the way.

**Comment by RayneRPT** [comment]
Now with version 5.5.3 crash to TV menu after the inicial page refreshs, was able to get it working one time, and now it doesn't work, crash every time. When it works sometimes can't recognize gamepad, I tried reinstalling and still there is this problem, well at least the better xcloud menu is working properly.

**Comment by redphx** [comment]
https://github.com/redphx/better-xcloud-tv/issues/7
